### PR TITLE
Fix interactive window hang by handling new type of exception

### PR DESCRIPTION
**Pull request by bschnurr**
fixes
https://github.com/microsoft/PTVS/issues/6853
https://github.com/microsoft/PTVS/issues/7118
https://github.com/microsoft/PTVS/issues/7219
https://github.com/microsoft/PTVS/issues/7064

### CODE REVIEW

File: Python/Product/PythonTools/ptvsd/repl/jupyter_client.py [hunk 1]
```diff
@@ -330,54 +330,53 @@ def flush(self):
...
-                        self.send_prompt('\n' + ps1, ps2, allow_multiple_statements=True)
-
-                    parent_id = m.parent_header['msg_id', None]
-                    if parent_id:
-                        on_reply = on_replies.pop((parent_id, msg_type), ())
-                        for callable in on_reply:
-                            callable(m)
-                except Empty:
-                    pass
+    try:
```
Comment (rchiodo) [inline Python/Product/PythonTools/ptvsd/repl/jupyter_client.py:333]: The indentation seems wrong? At least github is showing it that way

Comment (rchiodo) [reply Python/Product/PythonTools/ptvsd/repl/jupyter_client.py:333]: The try is at the same level as the 'def'.

How did you test that this worked?

Comment (bschnurr) [reply Python/Product/PythonTools/ptvsd/repl/jupyter_client.py:333]: oops. maybe I messed it up

**Review by rchiodo — APPROVED** [review]

File: Python/Product/PythonTools/ptvsd/repl/jupyter_client.py (hunk 1 above)
Comment (bschnurr) [reply Python/Product/PythonTools/ptvsd/repl/jupyter_client.py]: I was just manually testing

Comment (rchiodo) [reply Python/Product/PythonTools/ptvsd/repl/jupyter_client.py]: How did it work if the indent was off?

**Pull request merged**